test/e2e/telemeter_client_test.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 The Cluster Monitoring Operator Authors
```

Comment:
nit: 2023 :)

Reply:
Right, also the comment explaining the teste is wrong. Thanks